components/formats-gpl/src/loci/formats/in/MetamorphReader.java

```diff
@@ -190,17 +191,13 @@ public boolean isThisType(String name, boolean open) {
 
       while (baseName.indexOf('_') >= 0) {
         baseName = baseName.substring(0, baseName.lastIndexOf("_"));
-        if (checkSuffix(name, suffixes) &&
-          (new Location(parent, baseName + ".nd").exists() ||
-          new Location(parent, baseName + ".ND").exists()))
-        {
-          return true;
-        }
-        if (checkSuffix(name, suffixes) &&
-          (new Location(parent, baseName + ".htd").exists() ||
-          new Location(parent, baseName + ".HTD").exists()))
-        {
-          return false;
```

Comment:
Previously the behaviour in `isThisType` was to return `false` for the suffix htd, but now HTD has been added to `PARENT_INDEX_SUFFIX` it is returning `true` which is allowing it to identify with files which should be claimed by `CellWorxReader`.

Reply:
Oops, I just misread that as `return true` in the original code and overzealously refactored it. Fixed in latest commit.